### PR TITLE
Navigation update

### PR DESCRIPTION
**Pull request by lara-aigmueller**
- removed second anchor of primary nav items, updated examples and documentation

### CODE REVIEW

File: docs/_pages/test-pages/component-test-page.md [hunk 1]
```diff
@@ -17,162 +17,157 @@ type: test-page
...
-              <a class="nav__link nav__link--desktop" href="#">Why Dynatrace</a>
+              <a class="nav__link expandable__trigger" href="#noop">Why Dynatrace</a>
               <ul class="nav__list nav__list--secondary expandable__content">
                 <li class="nav__item">
-                  <a class="nav__link" href="#">Overview</a>
+                  <a class="nav__link" href="#noop">Overview</a>
                 </li>
                 <li class="nav__item">
-                  <a class="nav__link" href="#">Customer Stories</a>
+                  <a class="nav__link" href="#noop">Customer Stories</a>
```
Comment (tomheller) [inline docs/_pages/test-pages/component-test-page.md]: lowercase stories

File: docs/_pages/test-pages/component-test-page.md [hunk 2]
```diff
@@ -17,162 +17,157 @@ type: test-page
...
                 <li class="nav__item">
-                  <a class="nav__link" href="#">Contact</a>
+                  <a class="nav__link" href="#noop">Contact</a>
                 </li>
               </ul>
             </li>
             <li class="nav__item expandable expandable--nav is-current">
-              <a class="nav__link nav__link--mobile expandable__trigger" href="#">News & Resources</a>
-              <a class="nav__link nav__link--desktop" href="#">News & Resources</a>
+              <a class="nav__link expandable__trigger" href="#noop">News & Resources</a>
```
Comment (tomheller) [inline docs/_pages/test-pages/component-test-page.md]: lowercase resources

File: docs/_pages/test-pages/component-test-page.md [hunk 3]
```diff
@@ -17,162 +17,157 @@ type: test-page
...
-                  <a class="nav__link" href="#">Overview</a>
+                  <a class="nav__link" href="#noop">Overview</a>
                 </li>
                 <li class="nav__item">
-                  <a class="nav__link" href="#">Blog</a>
+                  <a class="nav__link" href="#noop">Blog</a>
                 </li>
                 <li class="nav__item">
-                  <a class="nav__link" href="#">Public Relations</a>
+                  <a class="nav__link" href="#noop">Public Relations</a>
```
Comment (tomheller) [inline docs/_pages/test-pages/component-test-page.md]: lowercase relations

File: src/expandable/expandable.js [hunk 4]
```diff
@@ -3,14 +3,16 @@ require('../js-common-components/accordion');
...
+
 const init = () => {
   $('.expandable__trigger').forEach(el => {
     if (!el.getAttribute('data-initialized')) {
-      el.addEventListener('click', (ev) => {
-        ev.preventDefault();
-        ev.stopPropagation();
-        el.closest('.expandable').classList.toggle(toggleClass);
-      });
+      el.addEventListener('click', handler(el));
```
Comment (tomheller) [inline src/expandable/expandable.js]: 👍

File: src/navbar/navbar.js [hunk 5]
```diff
@@ -111,4 +111,21 @@ const initData = () => {
...
 
-initData();
+const getSize = () => {
+  const size = window.getComputedStyle(document.body, ':after').getPropertyValue('content');
+  return size.replace(/[\W]/g, '');
+};
+
+const init = () => {
+  $('.expandable__trigger.nav__link').forEach(el => {
+    el.addEventListener('click', (ev) => {
```
Comment (tomheller) [inline src/navbar/navbar.js]: could also be not anonymous?

File: docs/_pages/changelog.md [hunk 6]
```diff
@@ -6,6 +6,14 @@ hideCode: true
 
 # Changelog
 
+<a name="3.0.0"></a>
+# [3.0.0](https://github.com/Dynatrace/css-groundhog/compare/v2.1.2...v3.0.0) (2017-08-18)
```
Comment (lara-aigmueller) [inline docs/_pages/changelog.md]: Yep, this should be done when merging v3.0.0 into master. Should I remove this link for now?

Comment (tomheller) [inline docs/_pages/changelog.md]: nah, keep it. just keep it in mind.

**Pull request merged**